docs/index.md

```diff
@@ -10,13 +11,13 @@ title: Introducing to you: TA<sup>2</sup>
 
 ## What is TA<sup>2</sup>?
 
-**Teaching Assistant's Assistant(TA<sup>2</sup>) is a desktop productivity application developed for teaching assistants to make their teaching journey smoother.**
+**Teaching Assistant's Assistant (TA<sup>2</sup>) is a desktop productivity application developed for teaching assistants to make their teaching journey smoother.**
 With TA<sup>2</sup>, you can manage student information and keep track of the status of their assignments. Just type in
 the corresponding commands and TA<sup>2</sup> will manage all these for you! No need to do all these manually!
 
 ## How is TA<sup>2</sup> different?
 
-TA<sup>2</sup> is optimised for use via a Command Line Interface(CLI) while still having the benefits of a Graphical User Interface(GUI). 
+TA<sup>2</sup> is optimised for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). 
```

Comment:
not sure which spacing format is better

Reply:
but if use, should be consistent for all develop or user guide

Reply:
yep, we should have space, visually more aesthetic